README.md

```diff
@@ -12,60 +12,84 @@ promotion from universe to main.
-
-```
-          Answered       +-7----------+     No feedback        +-9-------+
-      +------------------+ Incomplete +------------------------> Invalid |
-      |   old assign can | any        <--------------------+   | any     |
-      |   be kept as-is  +-----^------+                    |   +---------+
-      |                        |    question               |
-      |                        |                           |
-+-1---v-------+  Triage  +-2---+----+ Security needed  +-3-+------------------+
-|  New/Conf   +--------->+ New/Conf +----------------->+ New/Conf             |
-| unassigned  |          | assigned |                  | assigned to Security |
-+-------------+          +-+--+--+--+                  +-+--+--+--------------+
-                           |  |  |    +-8-----------+    |  |  |
-                           |  |  +----> Won't Fix   <----+  |  |
-                           |  |   Nack| any         | Nack  |  |
-                           |  |       +-------------+       |  |
-                           |  |                             |  |
-                           |  |       +-4-----------+       |  |
-                           |  +-------> In Progress <-------+  |
-                           |    Ack   | any         |  Ack     |
-                           |          +-------------+          |
-                           |             |Change that pulls    |
-                           |             |into Main            |
-+-6------------+           |         +-5-v-----------+         |
-| Fix Released |           +---------> Fix Committed <---------+
-| any          <---------------------+ any           |
-+--------------+  AA promotes        +---------------+
+## Process States
+
+This overview aims to clarify the meaning of MIR bug states, address previous 
+confusion, and indicate the next course of action on a MIR bug.
+
+```mermaid
+%% mermaid flowcharts documentation: https://mermaid.js.org/syntax/flowchart.html
+%%{ init: { 'flowchart': { 'curve': 'monotoneY' } } }%%
+flowchart TD 
+    subgraph New["New / Confirmed¹"]
+        Unassigned["<i>1.</i> Unassigned"]
+        Assigned["<i>2.</i> Assigned"]
+        AssignedToSecurity["<i>3.</i> Assigned to Security team"]
+        
+        Unassigned -->|"start triaging"| Assigned
+        Assigned -->|"if Security Review needed"| AssignedToSecurity
+    end
+
+    WontFix["<i>7.</i> Won't Fix"]:::FinalState
+
+    InProgress["<i>4.</i> In Progress"]
+    FixCommitted["<i>5.</i> Fix Committed"]
+    DependencyOrSeedChanges(["Dependency/Seed change that<br>pulls package(s) into <code>main</code>/<code>restricted</code>"])
+
+    FixReleased["<i>6.</i> Fix Released"]:::FinalState
+    Incomplete["<i>8.</i> Incomplete"]
+    Invalid["<i>9.</i> Invalid"]:::FinalState
+
+    New --->|"MIR team NACK"| WontFix
+    New --->|"MIR team ACK"| InProgress
+
+    New -.-|"<i>(unrelated)</i>"| DependencyOrSeedChanges 
```

Comment:
I do not see the point of this relation, could it be removed?

Reply:
I think that I misinterpreted the highlighted arrows. I will remove them. ![image](https://github.com/canonical/ubuntu-mir/assets/23220572/23ec7c0d-c2af-4920-bf5d-f469404b85a8)

Reply:
Thanks, resolved with [4bb6edd](https://github.com/canonical/ubuntu-mir/pull/23/commits/4bb6eddad4f6313289c27386b9ac6ab8bf81ad4e)